openshift/fluent-bit.cm.yaml

```diff
@@ -13,14 +13,13 @@ objects:
       name: fluent-bit-config
       namespace: ${NAMESPACE}
     data:
-      # Configuration files: server, input, filters and output
-      fluent-bit.conf: |
+      fluent-bit.conf: |-
         [SERVICE]
           Flush        5
           Daemon       Off
           # define the log format (see additional config map key/value)
           Parsers_File  parsers.conf
-          Log_Level    info
+          Log_Level    debug
```

Comment:
Let's keep this as info as we don't want too much verbosity during normal operations.

Reply:
👍